sections/shared/Layout/AppLayout/Footer/Footer.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC } from 'react';
+import styled from 'styled-components';
+
+const Footer: FC = () => {
+    const gitID = process.env.GIT_HASH_ID!.toString();
+
+    return (
+        <GitIDFooter>
+            {gitID}
+        </GitIDFooter>
+    );
+};
+
+const GitIDFooter = styled.div`
+	font-family: Akkurat LL TT;
```

Comment:
Pull this from our [fonts theme file](https://github.com/Kwenta/kwenta/blob/dev/styles/theme/fonts.ts) using `${props => props.theme.fonts.mono}`

You won't need the following `font-style` / `font-weight` as the resulting font should override.

Also I know mono font is a little different from the Figma designs, but I feel like it should be used for things like git hashes. cc @asherism